src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1077,6 +1077,17 @@ public static int getRpcListenBacklog()
         return conf.rpc_listen_backlog;
     }
 
+
+    public static long getInternodeConnectionTimeout()
+    {
+        return conf.internode_connect_timeout_in_ms;
+    }
+
+    public static long setInternodeConnectionTimeout(long timeoutInMillis)
+    {
+        return conf.internode_connect_timeout_in_ms = timeoutInMillis;
+    }
```

Comment:
If we plan on tweaking this during migrations, might be worth adding the capability to set this via nodetool

Reply:
from other comment: my current plan is for this to be the default and not change during migrations